docs/mobile_sdk.md

```diff
@@ -54,9 +65,8 @@ In previous example, you pass in the `myKey` and `myToken` you get from OpenWhis
 ```
 $ wsk property get --auth
 ```
-
 ```
-whisk auth              kkkkkkkk-kkkk-kkkk-kkkk-kkkkkkkkkkkk:tttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttt
+whisk auth        kkkkkkkk-kkkk-kkkk-kkkk-kkkkkkkkkkkk:tttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttt
```

Comment:
Should this be in the same formatting block as the command? To me, it looks like two separate things, rather than the return value of running a command.